src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -61,14 +60,14 @@ ci_lazy_methodst::ci_lazy_methodst(
 /// from the main entry point (usually provided with the --function command-
 /// line option
 /// \param symbol_table: global symbol table
-/// \param [out] lazy_methods: map from method names to relevant symbol and
+/// \param [out] method_bytecode: map from method names to relevant symbol and
```

Comment:
commit message is not very precise, consider changing `things` for `lazy_methods variable`

Reply:
It does change a couple of other things.